Sources/Orbit/Orbit.docc/Foundation/ScreenLayout.md

```diff
@@ -0,0 +1,28 @@
+# Screen Layout
+
+Wrap content in a unified screen layout to get consistently looking screens on iPhone or iPad of various sized.
+
+## Overview
+
+Screen layout gives content unified padding and frame behaviour that can be dynamic based on compact or regular width environment.
+
+## Adding screen layout to a content
+
+Use the `View.screenLayout()` modifier to add screen layout to a view:
+
+```swift
+VStack {
+    illustration
+    text
+    Spacer()
+    button
+}
+.screenLayout()
+```
+
+Screen layout modifier adds specified maximum width (maxReadableWidth by default) and padding behaviour for provided content.
+
+## Suppressing screen layout
+
+Child views can inspect the ``IsInsideScreenLayoutKey`` environment key to act upon this modifier. One example is a `Card` component that ignores horizontal paddings in `compact` environment.
```

Comment:
Isn't the key itself more of an implementation detail? When actually checking for this, we'll use the property on `EnvironmentValues`...